scripts/whitelist-fiscal-code-script/src/insert-fiscal-codes/insert-fiscal-codes.ts

```diff
@@ -0,0 +1,64 @@
+import { CosmosClient } from '@azure/cosmos';
+import * as CliProgress from 'cli-progress';
+import {
+  whitelistedFiscalCodeFileLogger,
+  notWhitelistedFiscalCodeFileLogger,
+  logger,
+} from '../utils/get-logger';
+
+export const insertFiscalCodes = async (
+  cosmosClient: CosmosClient,
+  fiscalCodes: string[],
+): Promise<void> => {
+  try {
+    const sleepTimeBetweenRequestsMs = Number(
+      process.env.SLEEP_TIME_BETWEEN_REQUESTS_MS ?? 500,
+    );
+    const databaseName = process.env.DATABASE_NAME as string;
+    const containerName = process.env.DATABASE_CONTAINER_NAME as string;
+
+    const database = cosmosClient.database(databaseName);
+    const container = database.container(containerName);
+
+    logger.info(`upserting ${fiscalCodes.length} fiscal codes...`);
+    const fiscalCodesSet = new Set(fiscalCodes);
+
+    const classicBar = new CliProgress.SingleBar(
+      {},
+      CliProgress.Presets.shades_classic,
+    );
+    classicBar.start(fiscalCodesSet.size, 0);
+
+    whitelistedFiscalCodeFileLogger.info('fiscalCode');
+    notWhitelistedFiscalCodeFileLogger.info('fiscalCode;message;stack');
```

Comment:
I would avoid making separate files. You can think of a single CSV with 3 columns, one for the timestamp, the other for the CF and finally a column with OK or NOT OK